.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      publishToMarketPlace:
+        description: "Publish to VS Code Marketplace ?"
+        required: true
+        type: choice
+        options:
+          - "true"
+          - "false"
+        default: "false"
+      publishToOVSX:
+        description: "Publish to OpenVSX Registry ?"
+        required: true
+        type: choice
+        options:
+          - "true"
+          - "false"
+        default: "false"
+jobs:
+  packaging-job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout vscode-knative
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: |
+          npm install -g typescript "vsce" "ovsx"
+          npm install
+          echo "EXT_VERSION=$(cat package.json | jq -r .version)" >> $GITHUB_ENV
+      - name: Build And Run Unit Tests
+        uses: coactions/setup-xvfb@b6b4fcfb9f5a895edadc3bc76318fae0ac17c8b3 #v1.0.1
+        with:
+          run: npm run test
+      - name: Run UI Tests
+        uses: coactions/setup-xvfb@b6b4fcfb9f5a895edadc3bc76318fae0ac17c8b3 #v1.0.1
+        with:
+          run: npm run base-ui-test
+      - name: Package
+        run: |
+          jq --tab '.extensionDependencies += [ "ms-kubernetes-tools.vscode-kubernetes-tools" ]' package.json  > package.json.new
+          mv package.json.new package.json
+          node ./out/build/update-readme.js
+          vsce package -o vscode-knative-${{ env.EXT_VERSION }}-${GITHUB_RUN_NUMBER}-${target}.vsix
+          sha256sum *-${target}.vsix > vscode-knative-${{ env.EXT_VERSION }}-${GITHUB_RUN_NUMBER}-${target}.vsix.sha256
+          ls -lash *.vsix *.sha256
+      - name: Upload VSIX Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: vscode-knative
+          path: vscode-knative*-${{ env.EXT_VERSION }}-${{github.run_number}}*.vsix
```

Comment:
I would remove the `*` from this pattern. It really should be just `vscode-knative-${{ env.EXT_VERSION }}-${{github.run_number}}.vsix`. That's the only artifact.